valohai_yaml/schema/param-item.yaml

```diff
@@ -1,3 +1,4 @@
+# Here is the full schema
```

Comment:
I'm not sure this comment has much value..?

Reply:
Leftover from my dialog with ChatGPT ;)